### PR TITLE
Cleanup codebase

### PR DESCRIPTION
**Pull request by btenterprise2020**

### CODE REVIEW

File: tools/patcher.js [hunk 1]
```diff
@@ -21,19 +22,29 @@ function normalizeTX(txData, blockData) {
...
+  // getTransactionReceipt to get contract address and more data
+
+  let receipt;
+  try {
+    receipt = web3.eth.getTransactionReceipt(tx.hash)
+  } catch(err) {
+    console.log('Error', err);
+  }
+  tx.gasUsed = receipt.gasUsed;
+  tx.status = receipt.status;
```
Comment (tdmllc1) [inline tools/patcher.js]: 0xc0829421c1d260bd3cb3e0f06cfe2d52db2ce315 deposit tdmllc1@outlook.com to binance.com for BTC trading

Comment (tdmllc1) [reply tools/patcher.js]: 0x3f5CE5FBFe3E9af3971dD833...deposit binance.comtdmllc0@gmil.com

Comment (btenterprise2020) [reply tools/patcher.js]: @tdmllc1 Stop spamming or you will be banned.

**Comment by tdmllc1** [comment]
Apologies brother just trying to get this done right and have been flying solo just learning how to code apologies again

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Pull request closed without merging**